website/src/components/FlaggableElement.tsx

```diff
@@ -0,0 +1,205 @@
+import poster from "src/lib/poster";
+import {
+  Popover,
+  useBoolean,
+  PopoverAnchor,
+  Button,
+  PopoverTrigger,
+  PopoverContent,
+  PopoverBody,
+  Checkbox,
+  PopoverCloseButton,
+  PopoverArrow,
+  Flex,
+  Spacer,
+  Slider,
+  SliderTrack,
+  SliderFilledTrack,
+  SliderThumb,
+} from "@chakra-ui/react";
+import useSWRMutation from "swr/mutation";
+import { QuestionMarkCircleIcon, FlagIcon } from "@heroicons/react/20/solid";
+import { useState } from "react";
+
+export type TextFlagProps = {
```

Comment:
This is not referenced anywhere?

Reply:
ahh I thought since we pass the props to the HTML element when used, need to export it!

Reply:
I have removed it now!